src/app/dashboard/sidebar/calculator-list/calculator-list.component.html

```diff
@@ -55,6 +55,7 @@
   <span class="sidebar-item" [routerLink]="['/calculators/process-heating-list']" routerLinkActive="selected">Process
     Heating</span>
   <div class="directory-item">
+    <span [routerLink]="['/calculators/wall-loss']" routerLinkActive="selected" class="sidebar-item">Wall Loss</span>
```

Comment:
Wall Loss and Flue Gas should be same order in sidebar and in the calculator list